### DISCUSSION TITLE
Error: Command failed: cd {{packageFileDir}}

### DISCUSSION THREAD

**Discussion by songkeys** [renovatebot/renovate#9065] [Request Help] [answered]
Not sure if I'm doing things wrong...

I'm following this thread: https://github.com/renovatebot/renovate/discussions/8274

In the `postUpgradeTasks.command` field, I wrote something like `cd {{packageFileDir}}` but it seems that the template is not parsed?

version: renovate@24.78.1

```
/bin/sh: 1: cd: can't cd to {{packageFileDir}}\n
```

**Comment by rarkins** [comment, marked as answer]
Did you set [`allowPostUpgradeCommandTemplating`](https://docs.renovatebot.com/self-hosted-configuration/#allowpostupgradecommandtemplating) to true in your bot config?

**Reply by songkeys** [reply]
You are right. I didn't. I was checking <https://docs.renovatebot.com/templates/>...

There're so many config fields that could easily get me lost. Anyway, thank you so much!!
